src/components/site/SiteNavigation.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropsTypes from "prop-types";
+import SiteNavButton from "./SiteNavButton";
+import CountySeal from "../foundation/CountySeal";
+
+const SiteNavLink = props => {
+  const { href, icon, text } = props;
+
+  return (
+    <li key={href}>
+      <a href={href}>
+        {icon && <i className={icon} aria-hidden="true"></i>}
+        {text}
+      </a>
+    </li>
+  );
+};
+
+const SiteNavigation = props => {
+  const { links = [] } = props;
+
+  return (
+    <nav id="bc_site-nav" className="site-nav" role="navigation">
+      <SiteNavButton />
+      <div className="bc_site-nav__list-container invisible">
+        <div className="text-center">
+          <a href="/">
+            <CountySeal className="bc_site-nav__seal" />
+          </a>
+        </div>
+        <ul id="bc_site-nav__links" className="bc_site-nav__list">
```

Comment:
are we still using className="bc_site_nav__list"? I did not find this className in our css file.

Reply:
@sgrg1 I will check it out.

Reply:
changed scss to use this class.